SplitFiles/slot.c

```diff
@@ -40,8 +40,8 @@ void slot_func(GameComponent* input, GameParameters* params) {
       state->bread = (BreadState*)params->message;
       state->power = TMAX;
       params->message = NULL;
-      //bit_beepfx(BEEPFX_CLANG);
-      params->effect = TUNE_EFFECT_DOWN; 
+      params->effect = TUNE_EFFECT_DOWN;
+      printf(PRINTAT"%c%c%c", state->xCoord + 7, state->yCoord -1, state->bread->type->letter);
```

Comment:
printf is quite slow, so we should probably replace this with screen charceter code
```suggestion
     *zx_pxy2saddr(state->xCoord + 6, state_>yCoord - 2) = state->bread->type->letter
```
(I think!)

Reply:
Errr.....I suppose this would run 8 times faster than printf given it's only writing one byte instead of the 8 required!

Putting it down to a late on a Sunday, ZX80 flashback.  Can feel you gearing up to start cranking out .asm files soon though ;-)